tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/deid/registered_cope_survey_suppression_test.py

```diff
@@ -1,11 +1,11 @@
 """
 Integration test to ensure records are properly sandboxed and dropped in the registered_cope_survey_suppression.py module.
 
-Removes any records that have an observation_source_concept_id as any of these values: 1310058, 1333325, 1333235,
- 1310065, 1333012, 1333234, 1332769, 702686, 1333156, 713888, 1333327, 1333118, 1310054, 1333326, 1310066, 596884,
-    596885, 596886, 596887, 596888, 596889, 1310137, 1310146, 1333015, 1333023, 1333016, 715714, 1310147, 715726.
+Removes any records that have an observation_source_concept_id as any of these values: 1310058, 1310065, 1333012,
+ 1333234, 702686, 1333327, 1333118, 1310054, 1333326, 1310066, 596884, 596885, 596886, 596887, 596888, 596889, 1310137,
+ 1310146, 1333015, 1333023, 1333016, 715714, 1310147, 715726.
 
-Original Issue: DC-1666
+Original Issue: DC-1666, DC-1740
 
 The intent is to ensure that no records exists that have any of the observation_source_concept_id above by sandboxing
 the rows and removing them from the observation table.
```

Comment:
@ksdkalluri Lines 78-106 set up the data for the test.  Update the comments in the setup to reflect these changes.